tests/py/dynamo/conversion/test_eq_aten.py

```diff
@@ -2,64 +2,65 @@
 import torch.nn as nn
 from parameterized import parameterized
 from torch.testing._internal.common_utils import run_tests
-from torch_tensorrt import Input
 
 from .harness import DispatchTestCase
 
 
 class TestEqualConverter(DispatchTestCase):
     @parameterized.expand(
         [
-            ("2d", (2, 1)),
-            ("3d", (2, 1, 2)),
+            ("2d", (5, 3)),
+            ("3d", (5, 3, 2)),
         ]
     )
-    def test_equal_tensor(self, _, shape):
-        class equal(nn.Module):
+    def test_eq_tensor(self, _, shape):
+        class eq(nn.Module):
             def forward(self, lhs_val, rhs_val):
                 return torch.ops.aten.eq.Tensor(lhs_val, rhs_val)
 
-        inputs = [torch.randn(shape), torch.randn(shape)]
+        inputs = [
+            torch.randint(0, 3, shape, dtype=torch.int32),
+            torch.randint(0, 3, shape, dtype=torch.int32),
+        ]
         self.run_test(
-            equal(),
+            eq(),
             inputs,
             output_dtypes=[torch.bool],
         )
 
     @parameterized.expand(
         [
-            ("2d", (2, 1), 1),
-            ("3d", (2, 1, 2), 2.0),
+            ("2d", (5, 3), 1),
+            ("3d", (5, 3, 2), 2.0),
         ]
     )
-    def test_equal_tensor_scalar(self, _, shape, scalar):
-        class equal(nn.Module):
```

Comment:
Is this case different from the previous `torch.ops.aten.eq.Tensor` case? As in in terms of the trace? Same application for the other `ge`, `gt`, `le`, `lt`, `ne`?

Reply:
I think `torch.ops.aten.eq.Tensor` and `torch.ops.aten.eq.Scalar` are different. The main difference is the second argument. In `torch.ops.aten.eq.Tensor`, it is `Tensor` while in `torch.ops.aten.eq.Scalar`, it's `Scalar`.